src/Workleap.OpenApi.MSBuild/IProcessWrapper.cs

```diff
@@ -4,5 +4,5 @@ namespace Workleap.OpenApi.MSBuild;
 
 public interface IProcessWrapper
 {
-    public Task<BufferedCommandResult> RunProcessAsync(string filename, string[] arguments, CancellationToken cancellationToken);
+    public Task<BufferedCommandResult> RunProcessAsync(string filename, string[] arguments, CancellationToken cancellationToken, Dictionary<string, string?>? envVars = null);
```

Comment:
Is it ok to make a breaking change on a public type? (it doesn't seem to have an external usage, so it may be ok)

Reply:
Good point, but now that you mention it this interface is indeed public, but I don't think it is meant to be used externally, so it could actually be switched to internal 🤔

Reply:
Endedup marking those interfaces to internal